.github/workflows/gradle-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Gradle Release
+
+#on:
+#  workflow_dispatch:
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: gradle
+
+    - run: ./gradlew clean app:assembleRelease
+
+    - name: Extract Version Code
+      run: |
+        echo "VERSION_CODE=$(grep -Eo 'versionCode\s+[0-9]+' app/build.gradle | grep -Eo '[0-9]+')" >> $GITHUB_ENV
+        echo "APP_ID=$(awk '/applicationId/ {gsub(/"/, "", $2); print $2; exit}' app/build.gradle)" >> $GITHUB_ENV
+        echo "Version code is: $VERSION_CODE"
+        echo "VERSION_NAME=$(awk '/versionName/ { gsub(/'\''/, "", $2); gsub(/-SNAPSHOT/, "", $2); print $2; exit }' app/build.gradle)" >> $GITHUB_ENV
```

Comment:
@tuancoltech What does this code do?:
```
gsub(/-SNAPSHOT/, "", $2);
```

Reply:
@jo-elimu It finds `-SNAPSHOT` in the second field of the `versionName` line, and remove it.
For instance, if we have:
`versionName '1.0.2-SNAPSHOT'`
then the output would be:
`'1.0.2'`

Reply:
@tuancoltech Removal of `-SNAPSHOT` should be handled by the Git tag creation step, not here.